src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/ValueDistFunction.java

```diff
@@ -95,8 +93,7 @@ public double setUtility(double currentdistance) {
 	@Override
 	public double getSubjectiveValue(LatLng objectiveData) {
 		if (interestlocation.containsKey(objectiveData) == false) {
-			LOGGER.error("Impossible to return the subjective value of the key " + objectiveData
-					+ " because the map doestn't contain this key.");
+			LOGGER.error("Impossible to return the subjective value, the map does not contain this key : ", objectiveData);
```

Comment:
Attention : 
cela va retourner 
`Impossible to return the subjective value, the map does not contain this key : `
sans la valeur voulue après 
cf [Doc slf4j](http://www.slf4j.org/manual.html#typical_usage)

Reply:
Fixed in commit ea9579e